### PR TITLE
[OYPD-252] Add Editor add menu with image scenario.

### PR DESCRIPTION
**Pull request by Kerby70**
Steps to review:
- Only test changes
- [ ] Tests pass.

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build956 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build956 |
| Installation process | http://install.openy-dev.ffwua.com/build956/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/956/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build956/profiles/contrib/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

**Review by alexschedrov — APPROVED** [review]
Code looks good.

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build959 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build959 |
| Installation process | http://install.openy-dev.ffwua.com/build959/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/959/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build959/profiles/contrib/openy/build/reports/behat

File: tests/features/menus_and_blocks/menus.feature [hunk 1]
```diff
@@ -14,16 +14,33 @@ Feature: Menus
...
+
+  Scenario: Create footer menu left item and check
+    Given I am logged in as a user with the Editor role
+    And I go to "/admin/structure/menu/manage/main/add"
+    And I fill in "Menu link title" with "BEHAT EDITOR ADD"
+    And I fill in "Link" with "http://openymca.org/"
+    And I check the box "Show as expanded"
+    And I attach the file "childcare.png" to "Icon image"
+    And I press "Save"
+    When I am an anonymous user
```
Comment (AlexNetman) [inline tests/features/menus_and_blocks/menus.feature:38]: Here we can go with selenium. And verify if menu items appear in dropdown with image displayed.

**Pull request merged**